docs/source/quick_start.rst

```diff
@@ -0,0 +1,199 @@
+============
+Quick Start
+============
+
+This guide is intended to jump start new Kokkos users.  Kokkos Core is the foundation for all other Kokkos Ecosystem projects, and is readily usable in your own for performance portability.
+
+
+-----------------------------
+Download Latest, Set Up Build 
+-----------------------------
+
+.. note::
+
+  Please become familiar with `Kokkos Requirements <https://kokkos.org/kokkos-core-wiki/requirements.html>`_, and verify that your machine has all necessary compilers, backend SDK and build system components.
+
+
+..
+ Nota bene:  the link for "Latest" will need to be updated for each release 
+..
+
+:bdg-link-primary:`Latest Release <https://github.com/kokkos/kokkos/releases/tag/4.2.01>`
+
+
+::
```

Comment:
One more thing I forgot is that you should use a syntax highlighting language here:

e.g.
```rst
.. code-block:: sh
```

See https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-code-block

Reply:
Done!